composer.json

```diff
@@ -13,7 +13,8 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "scriptfusion/static-class": "^1"
```

Comment:
> I understand this is a core package, however this is also a common pattern and it would be my intention to introduce this trait to the wider framework since the static utility class pattern is common enough to warrant this.

True, having a common package is useful here, and I even wrote one myself ( https://github.com/Roave/Dont ), and yes, it also uses traits.

 > I believe a private constructor is at least 3LOC if you respect PSR-2; 4LOC if you annotate that the empty constructor is intentional, as in [the trait](https://github.com/ScriptFUSION/StaticClass/blob/master/src/StaticClass.php)

Yes, that's still very tiny, and a non-functional change too. Better than a static reference to an external symbol, and an additional autoloading requirement.

 > I do not understand what you mean. `StaticClass` is tagged as 1.0.0 (stable). If this is not good enough what else do you need?

Adding a dependency just for this, where the dependency has a stability and maintenance guarantees lower than the ones in the framework (with all good intent and faith, it's a package maintained by a single developer, and not by the framework team) is a problem.

This is especially true when the package is at the top of the hierarchy (`zend-stdlib` is used [everywhere](https://packagist.org/packages/zendframework/zend-stdlib/dependents)).

Please don't take this as mistrust, I'm just telling you that the implications are bigger than the gains, and that by a lot.